tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/passes/ConfigurationPassImpl.scala

```diff
@@ -43,40 +45,72 @@ class ConfigurationPassImpl @Inject()(val options: PassOptions,
     * @return true, if the pass was successful
     */
   override def execute(): Boolean = {
-    var module = tlaModule.get
+    // try to read from the TLC configuration file
+    val configuredModule = tryConfigureFromTlcConfig(tlaModule.get)
 
+    // dump the configuration result
+    val outdir = options.getOrError("io", "outdir").asInstanceOf[Path]
+    PrettyWriter.write(configuredModule, new File(outdir.toFile, "out-config.tla"))
+
+    // set fallback options
```

Comment:
Suggested since it literally just repeats the name of the function, adding noise but no signal

```suggestion
```